Engine/ac/draw.cpp

```diff
@@ -210,87 +210,6 @@ void setpal() {
 
 #ifdef USE_15BIT_FIX
```

Comment:
probably worth getting rid of any use of USe_15BIT_FIX.

Reply:
I will remove it, but just to make things clear, USE_15BIT_FIX is always defined now, it was introduced when Chris Jones switched from the older version of Allegro (there is a comment in this source file above).
I am not sure whether the code under this @ifdef has any relation to the macro's original meaning though.